src/map/utils/zoneutils.cpp

```diff
@@ -333,7 +337,15 @@ namespace zoneutils
         }
 
         // handle npc spawn functions after they're all done loading
-        ForEachZone([](CZone* PZone) { PZone->ForEachNpc([](CNpcEntity* PNpc) { luautils::OnNpcSpawn(PNpc); }); });
+        // clang-format off
+        ForEachZone([](CZone* PZone)
```

Comment:
this type of this was the sole exception to brace on newline in the style guide. if we're dropping that the guide will need an update.

Reply:
clang-format chews up Lambdas in a nasty way.  Need to make sure the ones you fix are wrapped in // clang-format off/on blocks